doc/cephfs/administration.rst

```diff
@@ -234,6 +234,46 @@ For example, to only allow Nautilus clients, use:
 
 Clients running an older version will be automatically evicted.
 
+Enforcing minimum version of CephFS client is achieved by setting required
+client features. Commands to manipulate required client features of a file
+system:
+
+::
+
+    fs required_client_features cephfs add luminous
+    fs required_client_features cephfs rm luminous
```

Comment:
cephfs is the default FS name, I think, but we should probably make clear that it's the fs name in this example.